docs/source/download.rst

```diff
@@ -1,64 +1,103 @@
-=============
-Obtain Devito
-=============
-
-There are three main approaches to installing Devito. For those looking for the
-least-friction way to try Devito, we recommend the Docker route. For those
-looking to use Devito alongside other packages as part of a project, we support
-pip installation. If you're looking to develop for Devito, you might benefit from
-using the included conda environment that includes all the bells and whistles we
+===========================
+Download and Install Devito
+===========================
+
+There are three main approaches to installing Devito.
+
+For those looking for a quick “out-of-the-box” way to try Devito, we recommend
+the route of `Docker installation`_.
+For those interested in running on CPUs, please follow `Docker for CPUs`_.
+For those interested in deploying on GPUs, please follow `Docker for GPUs`_.
+For those looking to use Devito alongside other packages as part of a project, we support
+`pip installation`_. If you are looking to develop for Devito, you might benefit from
+following the `conda installation`_  that includes all the bells and whistles we
 recommend when developing for Devito.
 
+
 Docker installation
 -------------------
+
 You can install and run Devito via Docker_:
 
 .. _Docker: https://www.docker.com/  
 
+Docker for CPUs
+```````````````
+
 .. code-block:: shell
 
-   # get the code
-   git clone https://github.com/devitocodes/devito.git
-   cd devito
+   # NOTE (admin-sudo access may be needed) for some steps
+
+   # 1. Pull Devito image
+   docker pull devitocodes/devito:cpu-latest
 
-   # 1. run the tests
-   docker-compose run devito /tests
+   # 2. (Optional but recommended) Run the tests
+   docker run --rm --name testrun 'devitocodes/devito:cpu-latest' pytest tests/test_operator.py
 
-   # 2. start a jupyter notebook server on port 8888
-   docker-compose up devito
+   # 3. Start a Jupyter notebook server on port 8888
+   docker run --rm -it -p 8888:8888 devitocodes/devito:cpu-latest
 
-   # 3. start a bash shell with devito
-   docker-compose run devito /bin/bash
+   # 4. Start a bash shell with Devito
+   docker run --rm -it devitocodes/devito:cpu-latest /bin/bash
 
-The three sample commands above have only been included as illustrations of typical
+These sample commands above have only been included as illustrations of typical
 uses of Devito inside a container. They are not required to be run in that order.
+Please also review these commands within the dockerfile itself:
+`Dockerfile <https://github.com/devitocodes/devito/blob/master/docker/Dockerfile>`_
+
+
+Docker for GPUs
+```````````````
+
+.. code-block:: shell
+
+   # Requirements: Install `Nvidia container toolkit <https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html#setting-up-nvidia-container-toolkit>`_
```

Comment:
IMHO this should be emphasized more... it *is* a requirement and users must pay attention because without it everything will break. So I would at least pull it out the code-block